codegenerator/cli/npm/envio/src/bindings/Express.res

```diff
@@ -16,14 +20,18 @@ type middleware = (req, res, unit => unit) => unit
 @module("express") external jsonMiddleware: unit => middleware = "json"
 
 @send external use: (app, middleware) => unit = "use"
+@send external useFor: (app, string, middleware) => unit = "use"
```

Comment:
maybe we can use a labeled arg for the string? Had to check the docs for what this does